nims-notes-things/src/Form.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import TextArea from "./TextArea";
+import Input from "./Input";
+
+const Form = (props) => {
+  const [formData, setFormData] = useState({
+    name: "",
+    date: "",
+    note: "",
+  });
+
+  const [empty, setFormEmpty] = useState({
+    name: false,
+    date: false,
+    note: false,
+  });
+
+  const handleOnChange = (e, key) => {
+    const newForm = {
+      ...formData,
+      [key]: e.target.value,
+    };
+    setFormData(newForm);
+  };
+
+  const handleOnBlur = (e, key) => {
+    empty.map((data) => {
+      return <h2>key.data</h2>;
+    });
+  };
+
+  // const handleOnBlurName = e => {
+  //   if(formData.name === ''){
+  //     alert("This field is required")
+  //   }
+  // }
+  // const handleOnBlurDate = e => {
+  //   if(formData.name === ''){
+  //     alert("This field is required")
+  //   }
+  // }
+  // const handleOnBlurNote = (e,key) => {
+  //   if(formData.name === ''){
+  //     alert("This field is required")
+  //   }
+  // }
+  return (
+    <div className="App">
+      <Input
```

Comment:
You would put the error message somewhere here...
Depending on the state and if it's empty will this p tag display.
```
{empty.name && <p>This field is required</p>}
```